code/style.css

```diff
@@ -1,5 +1,360 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+html,
+body {
+  height: 100%;
+  font-family: helvetica, arial, sans-serif;
+  font-weight: 200;
+}
+
+header {
+  background-image: url(/img/header.jpg);
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  color: white;
+  width: 100%;
+  height: 100%;
+  min-height: 300px;
+  max-height: 500px;
+  box-sizing: border-box;
+}
+
+h1 {
+  font-size: 50px;
+  font-weight: 300;
+  margin: 0;
+}
+
+h2 {
+  font-weight: 100;
+  font-size: 30px;
+}
+
+p {
+  color: black;
+}
+
+
+.header-text {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px 0px 0px 20px;
+
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #fff;
+
+}
+
+nav {
+  background-color: #F6F4EF;
+  display: none;
+}
+
+nav a:link,
+a:visited {
+  color: rgb(150, 150, 150);
+  text-decoration: none;
+}
+
+nav a:hover,
+a:active {
+  color: #0B2D78;
+}
+
+nav .active {
+  color: #0B2D78;
+}
+
+nav ul {
+  margin: 20px 0px;
+  padding: 0;
+  list-style-type: none;
+  text-align: center;
+  font-size: 20px;
+  font-weight: 400;
+}
+
+nav li {
+  display: inline;
+  margin-right: 20px;
+}
+
+nav a {
+  color: rgb(150, 150, 150);
+  padding: 8px 10px;
+  text-decoration: none;
+  border-radius: 5px;
+}
 
 body {
```

Comment:
Usually you structure the css-file starting from the outside and top. So the body would usually go first, at the top of the document 😊